src/web/rustdoc.rs

```diff
@@ -762,6 +771,118 @@ mod test {
         })
     }
 
+    #[test]
+    fn redirect_latest_does_not_go_to_yanked_versions() {
+        wrapper(|env| {
+            let db = env.db();
+            db.fake_release()
+                .name("dummy")
+                .version("0.1.0")
+                .rustdoc_file("dummy/index.html", b"lah")
+                .create()
+                .unwrap();
```

Comment:
```suggestion
                .create()?;
```
This doesn't actually affect the test, but it makes it easier to read IMO